src/js/jsx/IconBar.jsx

```diff
@@ -74,12 +84,12 @@ define(function (require, exports, module) {
 
         componentDidMount: function () {
             this._updatePanelSizesDebounced = synchronization.debounce(this._updatePanelSizes, this, 500);
```

Comment:
The problem with that is that I have to remove the function later. Which means I need a reference to it in a different method.